src/plugins/intel_gpu/tests/functional/subgraph_tests/sdpa.cpp

```diff
@@ -63,6 +65,10 @@ class SDPA : virtual public ov::test::SubgraphBaseStaticTest {
 };
 
 TEST_F(SDPA, Inference) {
+    if (!tests::is_supported_sdpa_micro_kernel(ov::test::utils::DEVICE_GPU)) {
```

Comment:
can we reuse existing function?

Reply:
To use that function at the functional test, we should add more complexed to cmake(openvino_intel_gpu_graph) and source code(cldnn::engine).